docs/docs/FAQ/Assets-FAQ.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_position: 3
+---
+
+# Assets
+
+### Can I add my existing photo library?
+
+Yes, with an [external library](/docs/features/libraries.md).
+
+### What happens to existing files after I choose a new [Storage Template](/docs/administration/storage-template.mdx)?
+
+Template changes will only apply to new assets. To retroactively apply the template to previously uploaded assets, run the Storage Migration Job, available on the [Jobs](/docs/administration/jobs.md) page.
+
+### Why are only photos and not videos being uploaded to Immich?
+
+This often happens when using a reverse proxy or cloudflare tunnel in front of Immich. Make sure to set your reverse proxy to allow large POST requests. In `nginx`, set `client_max_body_size 50000M;` or similar. Also check the disk space of your reverse proxy, in some cases proxies caches requests to disk before passing them on, and if disk space runs out the request fails.
+
+### In the uploads folder, why are photos stored in the wrong date?
+
+This is fixed by running the storage migration job.
+
+### How can I backup data from Immich?
+
+See [backup and restore](/docs/administration/backup-and-restore.md).
+
+### Does Immich supports reading faces tags from the Exif?
+
+not at this moment
+
+### Does Immich support filtering of images defined as NSFW?
+
+As of now, this option is not implemented, but it seems that there is an [open discussion about it
+On Github](https://github.com/immich-app/immich/discussions/2451) you can submit a pull request or vote for the discussion
```

Comment:
Add a period before the "you", capitalize to "You" and add a period after "discussion".